e2e/features/filter-arrow-buttons-atom-basic.feature

```diff
@@ -0,0 +1,210 @@
+Feature: filter, arrow buttons and pop-up for atom basic table columns
+
+    Scenario: order is changed after clicking on Number cell title
```

Comment:
please, be consistent with other scenario names, common patter is

{example-name} is the same after ...